client_packages/LSOnline/Vehicle/VehicleEvents.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+mp.events.add({
+    "playerEnterVehicle": (vehicle, seat) => {
+        const engineStatus = vehicle.engine ? true : false;
```

Comment:
turning engine on should be implemented in another file.

Reply:
Thats prevent player to run automatically engine when he enters vehicle.